tests/fixtures/chacha20_poly1305.rs

```diff
@@ -0,0 +1,11 @@
+[
+    // https://tools.ietf.org/html/rfc8439#section-2.8.2
```

Comment:
It's possible to extract another test vector from https://datatracker.ietf.org/doc/html/rfc8439#appendix-A